### PR TITLE
WebTransportReceiveStreamStats Impl

### PR DESCRIPTION
**Pull request by KershawChang**

### CODE REVIEW

File: neqo-http3/src/lib.rs [hunk 1]
```diff
@@ -470,6 +473,11 @@ trait RecvStream: Stream {
     fn webtransport(&self) -> Option<Rc<RefCell<WebTransportSession>>> {
         None
     }
+
+    /// This function is only implemented by `WebTransportRecvStream`.
+    fn stats(&mut self, _conn: &mut Connection) -> Res<RecvStreamStats> {
+        Err(Error::Unavailable)
```
Comment (martinthomson) [inline neqo-http3/src/lib.rs:479]: It seems to me like you could just make this:

```rust
Ok(conn.recv_stream_stats(self.stream_id)?)
```

It's not like the stats provided would be invalid for non-webtransport streams, only that you wouldn't be reporting adjusted values.

File: neqo-transport/src/lib.rs [hunk 2]
```diff
@@ -26,7 +26,7 @@ mod path;
 mod qlog;
 mod quic_datagrams;
 mod recovery;
-mod recv_stream;
+pub mod recv_stream;
```
Comment (martinthomson) [inline neqo-transport/src/lib.rs:29]: I would prefer that you re-export the stats part.  That is,

```rust
mod recv_stream;
// ...
pub use recv_stream::RecvStreamStats;
```

There is stuff in the module that we don't need to make public, such as all the RecvStream stuff.

I know that we did that for send_stream, but that was probably also a mistake.

**Review by martinthomson — APPROVED** [review]
Two minor comments only.  Sorry for letting this drop.

File: neqo-http3/src/lib.rs (hunk 1 above)
Comment (KershawChang) [reply neqo-http3/src/lib.rs:479]: Unfortunately, `self` doesn't have `stream_id` here.
I'll keep returning an error here.

Comment (martinthomson) [reply neqo-http3/src/lib.rs:478]: Cool.  No worries.

**Pull request merged**